pkgs/development/python-modules/duckdb/default.nix

```diff
@@ -38,7 +38,6 @@ buildPythonPackage rec {
     '';
 
   env = {
-    BUILD_HTTPFS = 1;
```

Comment:
This is now managed in a separate repository, and I couldn't figure out how to get it to build under this new paradigm, so I just opted to avoid building it. It can still be used, but it won't be available out of the box anymore.